pkg/queue/request_metric_test.go

```diff
@@ -123,6 +126,52 @@ func TestRequestMetricHandlerPanickingHandler(t *testing.T) {
 	handler.ServeHTTP(resp, req)
 }
 
+func BenchmarkNewRequestMetricHandler(b *testing.B) {
+	var (
+		queueSizeMetric = stats.Int64(
```

Comment:
Ugh, this is unfortunate. We definitely need to refactor the queue-proxy to move all of these kinda things into a package rather than main... Not necessarily in this PR though.

Reply:
Yep same thing duplicated in many place 
ex: https://github.com/knative/serving/blob/master/pkg/activator/stats_reporter.go#L33

Reply:
I will refactor
Thank you :+1: